user/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from user.models import User
```

Comment:
```suggestion
from .models import User
```
にしましょう
ついでに並び替えと空行もお願いします！